content/en/methods/timelines.md

```diff
@@ -299,6 +299,58 @@ Invalid or missing Authorization header.
 
 ---
 
+## View link timeline {#link}
+
+```http
+GET /api/v1/timelines/link?url=:url HTTP/1.1
+```
+
+View public statuses containing a link to the specified currently-trending article. This only lists statuses from people who have opted in discoverability features.
```

Comment:
I am not a native speaker, but this feels like a word is missing.

> [...] This only lists statuses from people who have opted in **to** discoverability features.

seems more natural to me, but I might be totally wrong. @andypiper?

(An online dictionary I consulted actually proposed "for" instead of "to", but I have never ever seen this used before :confused:)